core/src/script/CGXP/plugins/MyPosition.js

```diff
@@ -61,6 +61,31 @@ cgxp.plugins.MyPosition = Ext.extend(gxp.plugins.Tool, {
      */
     recenteringZoom: 11,
 
+    /** api: config[drawAccuracy]
+     *  ``Boolean``
+     *  Condition for drawing the circle of accuracy, default is false. 
```

Comment:
Or maybe: "If true, a circle is displayed around the user's position, showing the accuracy of the geolocation. Default is false."